src/pudl/extract/epacems.py

```diff
@@ -51,89 +50,40 @@
     "Steam Load (1000 lb/hr)": "steam_load_1000_lbs",
     "SO2 Mass (lbs)": "so2_mass_lbs",
     "SO2 Mass Measure Indicator": "so2_mass_measurement_code",
-    "SO2 Rate (lbs/mmBtu)": "so2_rate_lbs_mmbtu",  # Not reading from CSV
-    "SO2 Rate Measure Indicator": "so2_rate_measure_flg",  # Not reading from CSV
-    "NOx Rate (lbs/mmBtu)": "nox_rate_lbs_mmbtu",  # Not reading from CSV
-    "NOx Rate Measure Indicator": "nox_rate_measurement_code",  # Not reading from CSV
     "NOx Mass (lbs)": "nox_mass_lbs",
     "NOx Mass Measure Indicator": "nox_mass_measurement_code",
     "CO2 Mass (short tons)": "co2_mass_tons",
     "CO2 Mass Measure Indicator": "co2_mass_measurement_code",
-    "CO2 Rate (short tons/mmBtu)": "co2_rate_tons_mmbtu",  # Not reading from CSV
-    "CO2 Rate Measure Indicator": "co2_rate_measure_flg",  # Not reading from CSV
     "Heat Input (mmBtu)": "heat_content_mmbtu",
     "Heat Input Measure Indicator": "heat_content_measure_flg",
-    "Primary Fuel Type": "primary_fuel_type",
-    "Secondary Fuel Type": "secondary_fuel_type",
-    "Unit Type": "unit_type",
-    "SO2 Controls": "so2_controls",
-    "NOx Controls": "nox_controls",
-    "PM Controls": "pm_controls",
-    "Hg Controls": "hg_controls",
-    "Program Code": "program_code",
 }
 """Dict: A dictionary containing EPA CEMS column names (keys) and replacement names to
 use when reading those columns into PUDL (values).
 
 There are some duplicate rename values because the column names change year to year.
 """
 
-# Any column that exactly matches one of these won't be read
-API_IGNORE_COLS = {
```

Comment:
I've switched to dropping all columns that aren't in `API_RENAME_DICT`, because we're only actually using columns we're renaming anyway.